### PR TITLE
Xrf mnt

### PR DESCRIPTION
**Pull request by tacaswell**
attn @licode

### CODE REVIEW

File: skxray/fitting/xrf_model.py [hunk 1]
```diff
@@ -399,19 +401,24 @@ def _add_area_param(self, element, constraint=None):
...
+            post_fix = "ma1_area"
+        else:
+            raise ValueError(
+                "{} is not a well formed element string".format(element))
+
+        param_name = '_'.join((base_element, post_fix))
 
         new_area = PARAM_DEFAULTS['area'].copy()
-        if constraint:
+        if constraint is not None:
```
Comment (licode) [inline skxray/fitting/xrf_model.py]: so "if constraint" is not clear enough?

Comment (tacaswell) [inline skxray/fitting/xrf_model.py]: `if constraint` will skip this block if the user passes in any 'falsy' value ( `[]`, `{}`, `''`, `0`, `False`) when what I think is meant here is 'if the user did not pass in a constraint, do not try to set one'.

Comment (licode) [inline skxray/fitting/xrf_model.py]: thanks. fair enough.

**Comment by licode** [comment]
Can you also push this again my new PR?

**Comment by tacaswell** [comment]
you can manually merge it into your branch.

**Comment by licode** [comment]
Manually added work from this pr to https://github.com/Nikea/scikit-xray/pull/275
So I close it.

**Pull request closed without merging**